tachyon/math/elliptic_curves/short_weierstrass/affine_point.h

```diff
@@ -32,9 +33,12 @@ class AffinePoint<_Curve, std::enable_if_t<_Curve::kIsSWCurve>>
 
   constexpr AffinePoint()
       : AffinePoint(BaseField::Zero(), BaseField::Zero(), true) {}
-  constexpr AffinePoint(const Point2<BaseField>& point, bool infinity = false)
+  constexpr AffinePoint(
+      const Point2<BaseField>& point,  // NOLINT(runtime/explicit)
+      bool infinity = false)
```

Comment:
Please add 2 constructors!

```suggestion
  constexpr explicit AffinePoint(const Point2<BaseField>& point);
  constexpr AffinePoint(const Point2<BaseField>& point, bool infinity)
````